docs/docs/docs/sbt/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+title: sbt plugin
+url: /docs/sbt
+---
+
+# What is `sbt-compendium`?
+
+`sbt-compendium` is a plugin that provides a client to communicate with [Compendium](https://github.com/higherkindness/compendium) service. It currently support:
+
+- Retrieval of protocols
+
+To start using `sbt-compendium`, add the following line in `project/plugin.sbt`:
+
+```
+addSbtPlugin("io.higherkindness" %% "sbt-compendium" % <version>)
+```
```

Comment:
We can do it in a following PR but `sbt-org-policies` supports replacing versions with the current one on each release. See `[comment]: # (Start Replace)` and `[comment]: # (End Replace)` here https://raw.githubusercontent.com/higherkindness/mu-scala/7f84033fc2de7c6f6a33462470fec70b4ac1cdbf/docs/src/main/docs/reference/modules-artifacts.md

Reply:
I would rather not depend on `sbt-org-policies` so it is easy to migrate out of it. But I think `sbt-mdoc` has a similar feature :thinking: I would look at it in another PR, sounds good?